doc/DISCLAIMER.md

```diff
@@ -1,12 +1,3 @@
-* Any known limitations, constrains or stuff not working, such as (but not limited to):
-    * requiring a full dedicated domain ?
-    * architectures not supported ?
-    * not-working single-sign on or LDAP integration ?
-    * the app requires an important amount of RAM / disk / .. to install or to work properly
-    * etc...
+## Configuration
 
-* Other infos that people should be aware of, such as:
-    * any specific step to perform after installing (such as manually finishing the install, specific admin credentials, ...)
-    * how to configure / administrate the application if it ain't obvious
-    * upgrade process / specificities / things to be aware of ?
-    * security considerations ?
+ * How to configure this app: From an admin panel at this address https://example.com/movim/?admin. Username and password are defined during installation.
```

Comment:
```suggestion
 * After installing the app, it can be configured at https://example.com/movim/?admin using the username and password defined during the installation.
```